util/cron/syncPerfGraphs.py

```diff
@@ -38,10 +38,43 @@ def main():
     logFile = args.logFile
     numRetries = args.numRetries
 
+    syncLocation = os.environ.get('SYNC_LOCATION', 'LOCAL')
```

Comment:
Where is this coming from? I'm not seeing this set in any of the scripts, is that something that will come later?

Reply:
It is in the CI changes in this PR: https://github.hpe.com/hpe/hpc-chapel-ci-config/pull/987.

Reply:
OK, I guess all I'd say then is maybe put it in the PR description? Like a link to that so others can follow. Looks good though.

Reply:
Added and merging :)